src/debug-macros.js

```diff
@@ -56,26 +37,13 @@ function getFlags(app, isDevelopingAddon) {
   if (_flags) {
     return _flags;
   }
-  const projectHelper = buildProjectHelper(app.project);
   let isProd = process.env.EMBER_ENV === 'production';
-  let dataPackage = projectHelper.getAddon('ember-data');
-  let storePackage = projectHelper.getAddon('@ember-data/store');
-  let version = dataPackage ? dataPackage.pkg.version : storePackage.pkg.version;
 
   let features;
   let packages;
-  let versions = {
-    GTE_VERSION_3_13: gte(version, '3.13.0'),
-  };
-  try {
-    features = app.project.require('@ember-data/private-build-infra/src/features')(isProd);
-  } catch (e) {
-    try {
-      features = app.project.require('@ember-data/-build-infra/src/features');
-    } catch (e) {
-      features = requireEsm('../addon/-infra/potential-features.js').default;
-    }
-  }
+
+  // >= 3.15.0.beta
+  features = app.project.require('@ember-data/private-build-infra/src/features')(isProd);
```

Comment:
@runspired what's the plan for making this public API?

Requiring addons to `require` `private-build-infra` to have features that depend on ED feature flags seems bad.

Reply:
@hjdivad there is no plan, but that's something we can maybe chat about at the F2F